packages/workspace/src/generators/ci-workflow/files/github/.github/workflows/__workflowFileName__.yml__tmpl__

```diff
@@ -18,16 +18,26 @@ jobs:
         with:
           filter: tree:0
           fetch-depth: 0
-
-      <% if(packageManager == 'pnpm'){ %>
-      - uses: pnpm/action-setup@v4
```

Comment:
pnpm still recommends using their action to setup so I think we should keep this.